src/pyfaf/actions/attach_centos_bugs.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2015  ABRT Team
+# Copyright (C) 2015  Red Hat, Inc.
+#
+# This file is part of faf.
+#
+# faf is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# faf is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with faf.  If not, see <http://www.gnu.org/licenses/>.
+
+from pyfaf.actions import Action
+from pyfaf.bugtrackers import bugtrackers
+from pyfaf.queries import (get_mantis_bug,
+                           get_report_by_hash,
+                           get_reportmantis,
+                           get_osrelease)
+from pyfaf.storage import ReportMantis
+
+
+class AttachCentosBugs(Action):
+    """
+    Looks up all bugs created using ABRT on CentOS MantisBT and attaches them
+    to reports using the bthash from URL. This is a workaround until proper
+    attaching is supported by the ABRT client.
```

Comment:
And if you didn't notice it also leads to unnecessary workarounds like this one that has to be done by someone else because someone didn't do his job.